src/settings/Settings.tsx

```diff
@@ -253,6 +253,13 @@ export const SETTINGS: { [setting: string]: ISetting } = {
         supportedLevels: LEVELS_FEATURE,
         default: false,
     },
+    "feature_custom_emotes": {
+        isFeature: true,
+        labsGroup: LabGroup.Messaging,
+        displayName: _td("Support custom emotes."),
```

Comment:
```suggestion
        displayName: _td("Support custom emotes"),
```

For whatever reason labs don't tend to be full sentences